.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: CI
 
-on: [push]
+on: [push, pull_request]
```

Comment:
push - triggers when people with access to repository push new branches or merges to master
pull_request - triggers for any PRs.